Makefile

```diff
@@ -43,8 +43,8 @@ draft-ietf-rats-eat.md: $(NC_COMMON_CDDL_FRAGS) \
 nc-cddl/%.cddl: cddl/%.cddl
 	mkdir -p nc-cddl
 	sed 's/;.*//' $< | \
-           sed -e 's/267/TBD/;s/268/TBD/;s/269/TBD/;s/270/TBD/;s/271/TBD/' | \
```

Comment:
For my build env, these lines have to be completely removed.

Reply:
I have commented them out,  but I don't see how the nc-cddl directory gets created by doing so.  I am still debugging other parts of my env to get the Makefile running, but leaving in those lines allowed for the nc-cddl directory to be created.

Reply:
Just remove the two sed lines that do the "TBD" substitutions.